plenum/server/node.py

```diff
@@ -132,6 +133,8 @@ def __init__(self,
         self.basedirpath = basedirpath or config.baseDir
         self.dataDir = self.config.nodeDataDir or "data/nodes"
 
+        self._view_change_timeout = self.config.VIEW_CHANGE_TIMEOUT
```

Comment:
Why dont we just have a timeout for election, so the pool cannot be in a leaderless state for the timeout, view-change is just a prequel to leader election